client/packages/requisitions/src/ResponseRequisition/api/api.ts

```diff
@@ -191,22 +191,37 @@ export const getResponseQueries = (sdk: Sdk, storeId: string) => ({
 
     throw new Error('Unable to update requisition');
   },
-  deleteLines: async (_: ResponseLineFragment[]) => {
-    // NOT YET IMPLEMENTED IN SERVER API
-
-    // const ids = responseLines.map(responseParser.toDeleteLine);
-    // const result = await sdk.deleteRequestLines({ ids, storeId });
-
-    // if (result.batchRequestRequisition.deleteRequestRequisitionLines) {
-    //   const failedLines =
-    //     result.batchRequestRequisition.deleteRequestRequisitionLines.filter(
-    //       line =>
-    //         line.response.__typename === 'DeleteRequestRequisitionLineError'
-    //     );
-    //   if (failedLines.length === 0) {
-    //     return result.batchRequestRequisition.deleteRequestRequisitionLines;
-    //   }
-    // }
+  deleteLine: async (responseLineId: string) => {
+    const ids = [{ id: responseLineId }];
+    const result = await sdk.deleteResponseLines({ ids, storeId });
+
+    if (result.batchResponseRequisition.deleteResponseRequisitionLines) {
+      const failedLines =
+        result.batchResponseRequisition.deleteResponseRequisitionLines.filter(
+          line =>
+            line.response.__typename === 'DeleteResponseRequisitionLineError'
+        );
+      if (failedLines.length === 0) {
+        return result.batchResponseRequisition.deleteResponseRequisitionLines;
+      }
+    }
```

Comment:
This uses the batch delete api even for singular.

Not currently used, but will be used for edit single line in future